src/index.tsx

```diff
@@ -43,6 +43,10 @@ let ensureTurnstile: () => Promise<any>;
   };
 }
 
+export function useTurnstile() {
+  return global.turnstile;
+}
+
```

Comment:
function already exists with a proper component reload if you werent 36 commits behind.